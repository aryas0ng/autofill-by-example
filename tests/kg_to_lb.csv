6,13.2
2,4.4
2,4.4
3,
10,
5,
4,
19,
2,
1,
9,
7,
4,
3,
12,
4,
1,
26,
8,
3,
3,
8,
4,