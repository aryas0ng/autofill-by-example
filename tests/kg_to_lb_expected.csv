6,13.2
2,4.4
2,4.4
3,6.6
10,22
5,11
4,8.8
19,41.8
2,4.4
1,2.2
9,19.8
7,15.4
4,8.8
3,6.6
12,26.4
4,8.8
1,2.2
26,57.2
8,17.6
3,6.6
3,6.6
8,17.6
4,8.8